### PR TITLE
String length 

### PR DESCRIPTION
**Pull request by fahdi**

### CODE REVIEW

File: stringlength.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	"fmt"
+	"strings"
+)
+
+func main() {
+	stringOne := "merpflakes"
+	stringTwo := "   "
+	stringThree := ""
+
+	if len(strings.TrimSpace(stringOne)) == 0 {
```
Comment (medinae) [inline stringlength.go:13]: 👍 
Could you create an isolated function to get the string length and try to unit test it ? I'm really curious about unit testing in Go !

Comment (fahdi) [reply stringlength.go:13]: I haven't tried it before, but lets do it :)

Comment (fahdi) [reply stringlength.go:13]: Its just using a built in function to demonstrate a simple len function, no unit test needed for built in function, that'd be an overkill.

Comment (medinae) [reply stringlength.go:13]: ok @fahdi. I will try to implement something testable with its unit tests this WE. Thanks dude 👍

**Pull request merged**